staking/types/validator.go

```diff
@@ -387,6 +387,32 @@ func (w *ValidatorWrapper) SanityCheck() error {
 	return nil
 }
 
+func (snapshot *ValidatorSnapshot) RawStake() numeric.Dec {
```

Comment:
Please add comment for this method. And use a better name for the method. e.g. StakePerSlot